public/main.js

```diff
@@ -0,0 +1,60 @@
+const randomNumber = Math.floor(Math.random() * (100) + 1)
+let counter = 0
+
+const guessNumber = () => {
+  const inputValue = document.getElementById("guessInput").value
+  if (inputValue < 1 || inputValue > 100) {
+    alert('Must be a number between 1 and 100')
+  } else if (inputValue == randomNumber) {
+    alert('You Win!')
+  } else if (inputValue > randomNumber) {
+    alert('Guess Lower')
+    counter++
+  } else if (inputValue < randomNumber) {
+    alert('Guess Higher')
+    counter++
+  }
+  document.getElementById("guesses").innerHTML = "Guesses so far: " + counter
+}
+
+const findIndex = () => {
+  const sentenceAsArray = document.getElementById("sentence").value.split(/\s*\b\s*/)
+  const indexElement = document.getElementById("indexNumber").value - 1
+  let wordAtIndex
+  if (indexElement < 0 || indexElement > sentenceAsArray.length) {
+    wordAtIndex = '_'
+  } else {
+    wordAtIndex = sentenceAsArray[indexElement]
+  }
+  document.getElementById("theWord").innerHTML = wordAtIndex + " is The Word"
+}
+
+const fibonacciDecrypt = () => {
+  const userInput = document.getElementById("fibonacciDecrypt").value
+  let fibSequence = []
+
+  fibSequence[0] = 0
+  fibSequence[1] = 1
```

Comment:
if you are going to define the indexes of an array immediately after defining the array, you might want to just define the array with the indexes you want. example instead of writing
 fibSequence =[] 
fibSequence[0] = 0 
fibSequence[1] = 1 
you might as well write 
fibSequence = [0,1]